pkg/scheduler/frameworkext/framework_extender.go

```diff
@@ -270,6 +283,8 @@ func (ext *frameworkExtenderImpl) RunPreBindPlugins(ctx context.Context, state *
 	reservation = reservation.DeepCopy()
 	reservation.Status.NodeName = nodeName
 
+	original := reservation
+	reservation = reservation.DeepCopy()
```

Comment:
Is there any way to avoid deepcopy the reservation twice?

Reply:
updated